demo/android/README.md

```diff
@@ -6,6 +6,12 @@ Rhino requires a valid Picovoice `AccessKey` at initialization. `AccessKey` acts
 You can get your `AccessKey` for free. Make sure to keep your `AccessKey` secret.
 Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get your `AccessKey`.
 
+## Setup
+
+1. Open the project in Android Studio
+2. Go to `Build > Select Build Variant...` and select the language you would like to run the demo in (e.g. enDebug -> English, itRelease -> Italian)
+3. Build and run on an installed simulator or a connected Android device
+
 ## Running the Demo
 
 Copy your AccessKey into the `ACCESS_KEY` variable in `MainActivity.java` before building the demo.
```

Comment:
the section below describing a default context no longer applies. Replace with something more generic